settings/acsf/post-install/post-install.php

```diff
@@ -15,6 +15,7 @@
 
 $site = $_ENV['AH_SITE_GROUP'];
 $env = $_ENV['AH_SITE_ENVIRONMENT'];
+$target_env = $site.$env;
```

Comment:
Given that there is no test coverage for this file, can we verify that this works?